CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -0,0 +1,81 @@
+- Change secondary network Pod controller to subscribe to CNIServer events to support bridging and VLAN network. ([#5767](https://github.com/antrea-io/antrea/pull/5767), [@jianjuns])
+- Use Antrea IPAM for Antrea native secondary network support. ([#5427](https://github.com/antrea-io/antrea/pull/5427), [@jianjuns])
+- Create different images for antrea-agent and antrea-controller to minimize the overall image size, speeding up the startup of both antrea-agent and antrea-controller. ([#5856](https://github.com/antrea-io/antrea/pull/5856) [#5902](https://github.com/antrea-io/antrea/pull/5902) [#5903](https://github.com/antrea-io/antrea/pull/5903), [@jainpulkit22])
+- Fix incorrect MTU configurations for the WireGuard encryption mode and GRE tunnel mode. ([#5880](https://github.com/antrea-io/antrea/pull/5880), [@hjiajing] [@tnqn])
+- Don't create tunnel interface (antrea-tun0) when using Wireguard encryption mode. ([#5885](https://github.com/antrea-io/antrea/pull/5885) [#5909](https://github.com/antrea-io/antrea/pull/5909), [@antoninbas])
+- Record an event when Egress IP assignment changes for better troubleshooting. ([#5765](https://github.com/antrea-io/antrea/pull/5765), [@jainpulkit22])
+- Delete OVS port and flows before releasing Pod IP. ([#5788](https://github.com/antrea-io/antrea/pull/5788), [@tnqn])
+- Update Windows documentation with more clearer installation guide and instructions. ([#5789](https://github.com/antrea-io/antrea/pull/5789), [@antoninbas])
+- Enable IPv4/IPv6 forwarding on demand automatically to eliminate the need for user intervention or dependencies on other components. ([#5833](https://github.com/antrea-io/antrea/pull/5833), [@tnqn])
+- Prioritize L7 NetworkPolicy flows over `TrafficControl` to avoid a potential issue that a `TrafficControl` CR with a redirect action to the same Pod could bypass the L7 engine. ([#5768](https://github.com/antrea-io/antrea/pull/5768), [@hongliangl])
+- Add ability to skip loading kernel modules in antrea-agent to support some specialized distributions (eg: Talos). ([#5754](https://github.com/antrea-io/antrea/pull/5754), [@antoninbas])
+- Add NetworkPolicy rule name in Traceflow observation. ([#5667](https://github.com/antrea-io/antrea/pull/5667), [@Atish-iaf])
+- Use Traceflow API v1beta1 to replace the deprecated API version in `antctl traceflow`. ([#5689](https://github.com/antrea-io/antrea/pull/5689), [@Atish-iaf])
+- Replace `net.IP` with `netip.Addr` in FlowExporter which optimizes the memory usage and improve the performance of the FlowExporter. ([#5532](https://github.com/antrea-io/antrea/pull/5532), [@antoninbas])
+- Update kubemark from v1.18.4 to v1.29.0 for antrea-agent-simulator. ([#5820](https://github.com/antrea-io/antrea/pull/5820), [@luolanzone])
+- Update Antrea ROADMAP.md document. ([#5807](https://github.com/antrea-io/antrea/pull/5807), [@antoninbas])
```

Comment:
Maybe we don't need to mention this in release notes?

Reply:
removed